dashboard/test/ui/features/acquisition_products/pd/teacher_application.feature

```diff
@@ -185,22 +150,83 @@ Scenario: Teacher saves, re-opens, and submits an application
 
   # Section 3
   Then I wait until element "h3" contains text "Section 3: Professional Learning Program Requirements"
-  Then I wait until element "input[name='committed']" is visible
-  And I press "input[name='committed']:first" using jQuery
-  And I press the first "input#understandFee" element
-  And I click selector "input[name='payFee']" if I see it
+  And I complete Section 3 of the teacher PD application
   And I press the first "button#next" element
 
   # Section 4
   Then I wait until element "h3" contains text "Section 4: Additional Demographic Information and Submission"
-  And I press "input[name='genderIdentity']:first" using jQuery
-  And I press the first "input[name='race']" element
-  And I press the first "input[name='howHeard']" element
-  And I press the first "input#agree" element
+  And I complete Section 4 of the teacher PD application
   And I press the first "button[type='submit']" element
 
   # Confirmation page
   Then I wait until element "h1" contains text "Thank you for submitting your application!"
   Then I see no difference for "Confirmation after submitting previously saved application"
   Then I close my eyes
-  
\ No newline at end of file
+
+Scenario: Teacher starts a new csp application and submits it
+  Given I create a teacher named "Severus"
+    And I am on "http://studio.code.org/pd/application/teacher"
```

Comment:
this indentation is weird -- I think this is what Meg was talking about but I'd lean towards assuming this indentation in the other scenarios was unintentional. But if it works, it's not a huge deal

Reply:
Good point, will change!